docs/developers-guide/devenv.md

```diff
@@ -204,7 +241,14 @@ By default, the tests only run against the `h2` driver. You can specify which dr
 DRIVERS=h2,postgres,mysql,mongo clojure -X:dev:drivers:drivers-dev:test
 ```
 
-Some drivers require additional environment variables when testing since they are impossible to run locally (such as Redshift and Bigquery). The tests will fail on launch and let you know what parameters to supply if needed.
+Some drivers require additional environment variables when testing since they are impossible to run locally (such as
+Redshift and Bigquery). The tests will fail on launch and let you know what parameters to supply if needed.
+
+If running tests from the REPL, you can call something like:
+
```

Comment:
It'd be nice to include a breadcrumb to find out what you need.

Suggested addition:
Most drivers need to be able to load some data (a few use static datasets) and all drivers need to be able to connect to an instance of that database. You can find out what is needed in each's drivers test data namespace which follows that pattern `metabase.test.data.<driver>`.

There should be an implementation of a multimethod `tx/dbdef->connection-details` which must produce a way to connect to a database. You can see what is required.

Here's the one for postgres in `metabase.test.data.postgres`:

```clojure
(defmethod tx/dbdef->connection-details :postgres
  [_ context {:keys [database-name]}]
  (merge
   {:host     (tx/db-test-env-var-or-throw :postgresql :host "localhost")
    :port     (tx/db-test-env-var-or-throw :postgresql :port 5432)
    :timezone :America/Los_Angeles}
   (when-let [user (tx/db-test-env-var :postgresql :user)]
     {:user user})
   (when-let [password (tx/db-test-env-var :postgresql :password)]
     {:password password})
   (when (= context :db)
     {:db database-name})))
```

You can see that this looks in the environment for:
- host (defaults to "localhost")
- port (defaults to 5432)
- user
- password

The function names indicate if they throw or not (although in this instance the ones that would throw are also supplied default values).

The `(tx/db-test-env-var :postgresql :password)` will look in the `env/env` map for `:mb-postgres-test-password` which will be set by the environmental variable `MB_POSTGRESQL_TEST_PASSWORD`

```clojure
(take 10 (keys environ.core/env))
(:mb-redshift-test-password
 :java-class-path
 :path
 :mb-athena-test-s3-staging-dir
 :iterm-profile
 :mb-snowflake-test-warehouse
 :mb-bigquery-cloud-sdk-test-service-account-json
 :tmpdir
 :mb-oracle-test-service-name
 :sun-management-compiler)
```